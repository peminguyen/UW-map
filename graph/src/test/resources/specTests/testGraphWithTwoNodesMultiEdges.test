CreateGraph graphWithTwoNodesMultiEdges

AddNode graphWithTwoNodesMultiEdges A
AddNode graphWithTwoNodesMultiEdges B
AddNode graphWithTwoNodesMultiEdges C

AddEdge graphWithTwoNodesMultiEdges C A eCA2
AddEdge graphWithTwoNodesMultiEdges C A eCA1

AddEdge graphWithTwoNodesMultiEdges A A eAA2
AddEdge graphWithTwoNodesMultiEdges A B eAB1
AddEdge graphWithTwoNodesMultiEdges A B eAB2

AddEdge graphWithTwoNodesMultiEdges B A eBA1
AddEdge graphWithTwoNodesMultiEdges B A eBA2
AddEdge graphWithTwoNodesMultiEdges B B eBB1
AddEdge graphWithTwoNodesMultiEdges B B eBB2

ListNodes graphWithTwoNodesMultiEdges
ListChildren graphWithTwoNodesMultiEdges A
ListChildren graphWithTwoNodesMultiEdges B
ListChildren graphWithTwoNodesMultiEdges C