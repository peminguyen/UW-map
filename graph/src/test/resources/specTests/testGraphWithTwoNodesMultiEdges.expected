created graph graphWithTwoNodesMultiEdges

added node A to graphWithTwoNodesMultiEdges
added node B to graphWithTwoNodesMultiEdges
added node C to graphWithTwoNodesMultiEdges

added edge eCA2 from C to A in graphWithTwoNodesMultiEdges
added edge eCA1 from C to A in graphWithTwoNodesMultiEdges

added edge eAA2 from A to A in graphWithTwoNodesMultiEdges
added edge eAB1 from A to B in graphWithTwoNodesMultiEdges
added edge eAB2 from A to B in graphWithTwoNodesMultiEdges

added edge eBA1 from B to A in graphWithTwoNodesMultiEdges
added edge eBA2 from B to A in graphWithTwoNodesMultiEdges
added edge eBB1 from B to B in graphWithTwoNodesMultiEdges
added edge eBB2 from B to B in graphWithTwoNodesMultiEdges

graphWithTwoNodesMultiEdges contains: A B C
the children of A in graphWithTwoNodesMultiEdges are: A(eAA2) B(eAB1) B(eAB2)
the children of B in graphWithTwoNodesMultiEdges are: A(eBA1) A(eBA2) B(eBB1) B(eBB2)
the children of C in graphWithTwoNodesMultiEdges are: A(eCA1) A(eCA2)